Assets/Scripts/GameController.cs

```diff
@@ -57,7 +55,7 @@ public class GameController : MonoBehaviour {
 		// Set startup variables
 		TestStartup();
 		// Prereq server variables and functions
-		TestServer();
+		//TestServer();
```

Comment:
I know this is commented out, but could you change the name to be something more descriptive? Had to scroll through the code on github to find out the consequences of commenting this out.  Is it testing the server *connection*?_